.travis.yml

```diff
@@ -88,6 +88,7 @@ before_install:
         popd;
         export PATH="/usr/local/opt/ccache/libexec:$PATH";
         git clone https://github.com/frankmorgner/OpenSCToken.git;
+        sudo rm -rf /Library/Developer/CommandLineTools
```

Comment:
Why do we need to remove the commandline tools?

Reply:
Because they are not uptodate in travis? See my previous comment

/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/sys/cdefs.h:807:2: error: Unsupported architecture

#error Unsupported architecture

 ^

Reply:
I see. I assumed that XCode updates this automatically...